app/assets/javascripts/rooms.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function(){
```

Comment:
this code is gonna run on every view, we should restrict it to those views who actually have a map